Source/WebCore/rendering/LegacyLineLayout.cpp

```diff
@@ -728,9 +728,9 @@ static inline ExpansionBehavior expansionBehaviorForInlineTextBox(RenderBlockFlo
 {
     // Tatechuyoko is modeled as the Object Replacement Character (U+FFFC), which can never have expansion opportunities inside nor intrinsically adjacent to it.
     if (textBox.renderer().style().textCombine() == TextCombine::All)
-        return ForbidLeftExpansion | ForbidRightExpansion;
+        return ExpansionBehavior();
```

Comment:
I think it's valuable to not delete the word "forbid" here. You want the reader of this code to know what the behavior is without having to click through to see what the constructor does.